apps/clinical/src/pages/list/styles/index.module.scss

```
@@ -0,0 +1,20 @@
+.page {
+  padding-top: $spacing-10;
+  padding-inline: calc($spacing-12 + $spacing-05);
+}
+
+.tabs {
+  box-shadow: 0 3px 6px 0 $shadow;
+}
+
+.message {
+  @include type-style('body-compact-01');
+  color: $support-error;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)